slsim/Deflectors/DeflectorTypes/epl_sersic.py

```diff
@@ -69,8 +69,11 @@ def light_model_lenstronomy(self, band=None):
 
     @property
     def halo_properties(self):
-        """Properties of the Halo (no halo is defined here)
+        """Mass density logarithmic slope.
 
-        :return: None
+        :return: gamma (with =2 is isothermal)
         """
-        return None
+        try:
+            return float(self._deflector_dict["gamma_pl"])
+        except:
```

Comment:
This is causing pre-commit fail. I think need to specify what are the expected exceptions?